R/oauth-flow-auth-code.R

```diff
@@ -85,14 +87,18 @@ req_oauth_auth_code <- function(req, client,
 #' Section 4.1. This is the most commonly used OAuth flow where the user is
 #' opens a page in their browser, approves the access, and then returns to R.
 #'
-#' `oauth_flow_auth_code()` is a high-level wrapper that should
-#' work with APIs that adhere relatively closely to the spec. The remaining
-#' low-level functions can be used to assemble a custom flow for APIs that are
-#' further from the spec:
+#' `oauth_flow_auth_code()` is a high-level wrapper that should work with APIs
+#' that adhere relatively closely to the spec. When possible, it redirects users
+#' back to a temporary local webserver to capture the authorization code. When
+#' this is not possible -- for example, when running on a hosted platform like
+#' RStudio Server -- it prompts the user to enter the code manually instead.
+#'
+#' The remaining low-level functions can be used to assemble a custom flow for
+#' APIs that are further from the spec:
 #'
 #' * `oauth_flow_auth_code_url()` generates the url where the user is sent.
-#' * `oauth_flow_auth_code_listen()` starts an webserver that listens for
-#'   the response from the resource server.
+#' * `oauth_flow_auth_code_listen()` starts a temporary local webserver that
```

Comment:
I can't comment on the line just above because it's not in the diff, but it's another example of potentially confusing phrasing about "the user".

Reply:
Updated the wording.